README.md

```diff
@@ -140,3 +140,11 @@ yarn build:web
 The command `yarn release` helps to automate the release process.
 
 Run the command and follow the prompts.
+
+## MixPanel
+
+MixPanel is used for non-private deployments of the interaface.
```

Comment:
```suggestion
MixPanel is used for non-private deployments of the [userinyerface](https://userinyerface.com).
```

Reply:
or that one
```suggestion
MixPanel is used for non-private deployments of the interface.
```

Reply:
Good catch! https://github.com/shapeshift/web/pull/5926/commits/8dc421f4c837ce4f8851fc49bb8e1034f660d1a4